docs/.pre-commit-config.yaml

```
@@ -0,0 +1,46 @@
+---
+repos:
+  # import formatter with black configurations
+    - repo: https://github.com/pycqa/isort
+      rev: 5.10.1
+      hooks:
+          - id: isort
+            name: isort (python)
+            args: [--profile, black, --filter-files]
+
+  # Code formatter for both python files and jupyter notebooks
+    - repo: https://github.com/psf/black
```

Consider revising the comment to match the indentation of the line below.
```suggestion
    # Code formatter for both python files and jupyter notebooks
    - repo: https://github.com/psf/black
```